src/frontend/hooks/google-auth.js

```diff
@@ -0,0 +1,76 @@
+import { useCallback } from 'react';
+import { useGoogleLogin } from 'react-google-login';
+import * as userActions from '../state/user/actions';
+import { useDispatch } from 'react-redux';
+import api from '../api';
+import { useGoogleLogout } from 'react-google-login';
+import Cookies from 'js-cookie';
+
+// TODO: store token in localStorage.
```

Comment:
Is there still a need to worry about this ? I thought it was being stored inside the cookie.